Support Nuvoton's new target NUMAKER_IOT_M487

ccli8 opened the pull request:
### Description

This PR adds support for Nuvoton's new target **NUMAKER_IOT_M487**, which is a branched version of existed **NUMAKER_PFM_M487** with change on on-board modules.

### Pull request type

    [ ] Fix
    [ ] Refactor
    [X] Target update
    [ ] Functionality change
    [ ] Breaking change

ccli8 committed:
Support Nuvoton's NuMaker-IoT-M487 board

ccli8 committed:
[M487] Add missing mbedtls_device.h for NUMAKER_IOT_M487

ccli8 commented:
### Greentea/ARM

```
mbedgt: test suite report:
+----------------------+------------------+--------------------------------------------------------------------------------------+--------+--------------------+-------------+
| target               | platform_name    | test suite                                                                           | result | elapsed_time (sec) | copy_method |
+----------------------+------------------+--------------------------------------------------------------------------------------+--------+--------------------+-------------+
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-device_key-tests-device_key-functionality                           | OK     | 22.92              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-basic_test                        | OK     | 17.09              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-basic_test_default                | OK     | 17.58              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_async_validate               | OK     | 19.28              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_control_async                | OK     | 24.8               | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_control_repeat               | OK     | 20.04              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_selection                    | OK     | 17.68              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_setup_failure                | OK     | 18.25              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_teardown_failure             | OK     | 17.95              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-control_type                      | OK     | 18.56              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-minimal_async_scheduler           | OK     | 18.63              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-minimal_scheduler                 | OK     | 19.19              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-test_assertion_failure_test_setup | OK     | 17.2               | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-test_setup_case_selection_failure | OK     | 17.24              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-test_setup_failure                | OK     | 17.15              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-test_skip                         | OK     | 17.2               | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-events-queue                                                           | OK     | 25.24              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-events-timing                                                          | OK     | 77.95              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-integration-basic                                                      | OK     | 16.58              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-c_strings                                                 | OK     | 19.29              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-crc                                                       | OK     | 18.93              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-dev_null                                                  | OK     | 19.0               | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-echo                                                      | OK     | 19.12              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-flashiap                                                  | OK     | 20.24              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-generic_tests                                             | OK     | 17.92              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-lp_ticker                                                 | OK     | 23.96              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-lp_timeout                                                | OK     | 60.39              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-lp_timer                                                  | OK     | 22.46              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-race_test                                                 | OK     | 19.05              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-rtc                                                       | OK     | 32.49              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-sleep_lock                                                | OK     | 17.31              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-stl_features                                              | OK     | 18.32              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-ticker                                                    | OK     | 43.87              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-timeout                                                   | OK     | 57.43              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-timer                                                     | OK     | 24.66              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-timerevent                                                | OK     | 19.5               | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_functional-callback                                               | OK     | 18.57              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_functional-callback_big                                           | OK     | 19.5               | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_functional-callback_small                                         | OK     | 18.77              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_functional-functionpointer                                        | OK     | 17.32              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-common_tickers                                                | OK     | 34.99              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-common_tickers_freq                                           | OK     | 37.41              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-critical_section                                              | OK     | 17.38              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-flash                                                         | OK     | 21.27              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-lp_ticker                                                     | OK     | 18.28              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-rtc                                                           | OK     | 60.71              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-rtc_reset                                                     | OK     | 21.79              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-rtc_time                                                      | OK     | 18.84              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-rtc_time_conv                                                 | OK     | 36.08              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-sleep                                                         | OK     | 20.69              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-sleep_manager                                                 | OK     | 17.44              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-sleep_manager_racecondition                                   | OK     | 30.18              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-ticker                                                        | OK     | 29.47              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-us_ticker                                                     | OK     | 17.61              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-critical_section                                         | OK     | 19.29              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-error_handling                                           | OK     | 18.54              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-filehandle                                               | OK     | 18.48              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-sharedptr                                                | OK     | 17.86              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-singletonptr                                             | OK     | 17.3               | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-system_reset                                             | OK     | 18.25              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-transaction                                              | OK     | 18.06              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-attributes                                              | OK     | 19.07              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-call_before_main                                        | OK     | 17.02              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-cpp                                                     | OK     | 16.92              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-div                                                     | OK     | 17.04              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-static_assert                                           | OK     | 17.65              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-basic                                              | OK     | 26.98              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-circularbuffer                                     | OK     | 24.19              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-condition_variable                                 | OK     | 21.02              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-event_flags                                        | OK     | 19.94              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack                                     | OK     | 18.34              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-kernel_tick_count                                  | OK     | 20.03              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-mail                                               | OK     | 21.66              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-malloc                                             | OK     | 38.01              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-memorypool                                         | OK     | 24.62              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-mutex                                              | OK     | 21.51              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-queue                                              | OK     | 20.94              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-rtostimer                                          | OK     | 19.91              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-semaphore                                          | OK     | 22.11              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-signals                                            | OK     | 24.15              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-threads                                            | OK     | 29.12              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedtls-multi                                                          | OK     | 18.56              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedtls-selftest                                                       | OK     | 19.0               | default     |
+----------------------+------------------+--------------------------------------------------------------------------------------+--------+--------------------+-------------+
mbedgt: test suite results: 83 OK
```

ccli8 commented:
### Greentea/GCC_ARM

```
mbedgt: test suite report:
+--------------------------+------------------+--------------------------------------------------------------------------------------+--------+--------------------+-------------+
| target                   | platform_name    | test suite                                                                           | result | elapsed_time (sec) | copy_method |
+--------------------------+------------------+--------------------------------------------------------------------------------------+--------+--------------------+-------------+
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-device_key-tests-device_key-functionality                           | OK     | 22.83              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-basic_test                        | OK     | 17.63              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-basic_test_default                | OK     | 17.33              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_async_validate               | OK     | 19.17              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_control_async                | OK     | 24.78              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_control_repeat               | OK     | 19.31              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_selection                    | OK     | 17.52              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_setup_failure                | OK     | 18.1               | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_teardown_failure             | OK     | 17.83              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-control_type                      | OK     | 19.31              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-minimal_async_scheduler           | OK     | 18.71              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-minimal_scheduler                 | OK     | 18.95              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-test_assertion_failure_test_setup | OK     | 16.64              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-test_setup_case_selection_failure | OK     | 17.27              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-test_setup_failure                | OK     | 17.78              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-test_skip                         | OK     | 17.88              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-events-queue                                                           | OK     | 24.61              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-events-timing                                                          | OK     | 78.49              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-integration-basic                                                      | OK     | 17.17              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-c_strings                                                 | OK     | 19.06              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-crc                                                       | OK     | 18.24              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-dev_null                                                  | OK     | 22.55              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-echo                                                      | OK     | 19.6               | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-flashiap                                                  | OK     | 19.92              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-generic_tests                                             | OK     | 17.83              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-lp_ticker                                                 | OK     | 23.17              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-lp_timeout                                                | OK     | 62.66              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-lp_timer                                                  | OK     | 22.41              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-race_test                                                 | OK     | 18.61              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-rtc                                                       | OK     | 32.77              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-sleep_lock                                                | OK     | 17.3               | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-stl_features                                              | OK     | 17.62              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-ticker                                                    | OK     | 46.03              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-timeout                                                   | OK     | 58.35              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-timer                                                     | OK     | 24.69              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-timerevent                                                | OK     | 18.61              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_functional-callback                                               | OK     | 18.84              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_functional-callback_big                                           | OK     | 18.81              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_functional-callback_small                                         | OK     | 18.73              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_functional-functionpointer                                        | OK     | 17.72              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-common_tickers                                                | OK     | 35.7               | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-common_tickers_freq                                           | OK     | 40.62              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-critical_section                                              | OK     | 17.26              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-flash                                                         | OK     | 18.05              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-lp_ticker                                                     | OK     | 17.58              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-rtc                                                           | OK     | 60.42              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-rtc_reset                                                     | OK     | 22.48              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-rtc_time                                                      | OK     | 19.39              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-rtc_time_conv                                                 | OK     | 42.38              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-sleep                                                         | OK     | 19.82              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-sleep_manager                                                 | OK     | 17.03              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-sleep_manager_racecondition                                   | OK     | 30.17              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-ticker                                                        | OK     | 29.26              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-us_ticker                                                     | OK     | 16.77              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-critical_section                                         | OK     | 19.13              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-error_handling                                           | OK     | 18.0               | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-filehandle                                               | OK     | 19.09              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-sharedptr                                                | OK     | 17.7               | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-singletonptr                                             | OK     | 17.19              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-system_reset                                             | OK     | 17.64              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-transaction                                              | OK     | 18.08              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-attributes                                              | OK     | 19.63              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-call_before_main                                        | OK     | 16.72              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-cpp                                                     | OK     | 16.44              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-div                                                     | OK     | 16.7               | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-static_assert                                           | OK     | 16.63              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-basic                                              | OK     | 28.69              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-circularbuffer                                     | OK     | 23.2               | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-condition_variable                                 | OK     | 17.14              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-event_flags                                        | OK     | 19.86              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack                                     | OK     | 18.31              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-kernel_tick_count                                  | OK     | 19.84              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-mail                                               | OK     | 22.02              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-malloc                                             | OK     | 38.78              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-memorypool                                         | OK     | 25.08              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-mutex                                              | OK     | 21.55              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-queue                                              | OK     | 20.57              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-rtostimer                                          | OK     | 19.5               | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-semaphore                                          | OK     | 22.06              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-signals                                            | OK     | 23.75              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-threads                                            | OK     | 29.27              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedtls-multi                                                          | OK     | 19.76              | default     |
| NUMAKER_IOT_M487-GCC_ARM | NUMAKER_IOT_M487 | mbed-os-tests-mbedtls-selftest                                                       | OK     | 19.73              | default     |
+--------------------------+------------------+--------------------------------------------------------------------------------------+--------+--------------------+-------------+
mbedgt: test suite results: 83 OK
```

ccli8 commented:
### Greentea/IAR

```
mbedgt: test suite report:
+----------------------+------------------+--------------------------------------------------------------------------------------+--------+--------------------+-------------+
| target               | platform_name    | test suite                                                                           | result | elapsed_time (sec) | copy_method |
+----------------------+------------------+--------------------------------------------------------------------------------------+--------+--------------------+-------------+
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-device_key-tests-device_key-functionality                           | OK     | 22.13              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-basic_test                        | OK     | 17.3               | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-basic_test_default                | OK     | 17.25              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_async_validate               | OK     | 18.67              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_control_async                | OK     | 24.97              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_control_repeat               | OK     | 18.96              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_selection                    | OK     | 16.62              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_setup_failure                | OK     | 17.63              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-case_teardown_failure             | OK     | 18.75              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-control_type                      | OK     | 17.92              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-minimal_async_scheduler           | OK     | 27.72              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-minimal_scheduler                 | OK     | 18.58              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-test_assertion_failure_test_setup | OK     | 16.42              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-test_setup_case_selection_failure | OK     | 21.2               | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-test_setup_failure                | OK     | 16.72              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-features-frameworks-utest-tests-unit_tests-test_skip                         | OK     | 17.03              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-events-queue                                                           | OK     | 24.73              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-events-timing                                                          | OK     | 77.13              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-integration-basic                                                      | OK     | 15.56              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-c_strings                                                 | OK     | 18.72              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-crc                                                       | OK     | 17.67              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-dev_null                                                  | OK     | 17.91              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-echo                                                      | OK     | 19.03              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-flashiap                                                  | OK     | 19.48              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-generic_tests                                             | OK     | 17.59              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-lp_ticker                                                 | OK     | 22.74              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-lp_timeout                                                | OK     | 62.25              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-lp_timer                                                  | OK     | 21.63              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-race_test                                                 | OK     | 19.16              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-rtc                                                       | OK     | 32.5               | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-sleep_lock                                                | OK     | 16.47              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-stl_features                                              | OK     | 18.31              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-ticker                                                    | OK     | 45.67              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-timeout                                                   | OK     | 59.66              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-timer                                                     | OK     | 24.51              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_drivers-timerevent                                                | OK     | 18.31              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_functional-callback                                               | OK     | 21.44              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_functional-callback_big                                           | OK     | 18.17              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_functional-callback_small                                         | OK     | 22.0               | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_functional-functionpointer                                        | OK     | 16.7               | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-common_tickers                                                | OK     | 35.27              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-common_tickers_freq                                           | OK     | 40.14              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-critical_section                                              | OK     | 17.08              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-flash                                                         | OK     | 18.08              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-lp_ticker                                                     | OK     | 17.41              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-rtc                                                           | OK     | 60.14              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-rtc_reset                                                     | OK     | 24.42              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-rtc_time                                                      | OK     | 18.68              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-rtc_time_conv                                                 | OK     | 42.73              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-sleep                                                         | OK     | 19.16              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-sleep_manager                                                 | OK     | 16.66              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-sleep_manager_racecondition                                   | OK     | 29.47              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-ticker                                                        | OK     | 29.31              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_hal-us_ticker                                                     | OK     | 16.8               | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-critical_section                                         | OK     | 18.97              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-error_handling                                           | OK     | 17.64              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-filehandle                                               | OK     | 17.83              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-sharedptr                                                | OK     | 19.14              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-singletonptr                                             | OK     | 16.67              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-system_reset                                             | OK     | 17.53              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbed_platform-transaction                                              | OK     | 21.31              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-attributes                                              | OK     | 18.39              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-call_before_main                                        | OK     | 16.09              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-cpp                                                     | OK     | 16.75              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-div                                                     | OK     | 16.61              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-mbed-static_assert                                           | OK     | 16.55              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-basic                                              | OK     | 28.28              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-circularbuffer                                     | OK     | 23.05              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-condition_variable                                 | OK     | 17.16              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-event_flags                                        | OK     | 19.7               | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-heap_and_stack                                     | OK     | 17.33              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-kernel_tick_count                                  | OK     | 19.24              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-mail                                               | OK     | 21.23              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-malloc                                             | OK     | 37.25              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-memorypool                                         | OK     | 24.23              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-mutex                                              | OK     | 25.28              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-queue                                              | OK     | 20.36              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-rtostimer                                          | OK     | 19.58              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-semaphore                                          | OK     | 21.52              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-signals                                            | OK     | 23.74              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedmicro-rtos-mbed-threads                                            | OK     | 28.95              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedtls-multi                                                          | OK     | 17.95              | default     |
| NUMAKER_IOT_M487-IAR | NUMAKER_IOT_M487 | mbed-os-tests-mbedtls-selftest                                                       | OK     | 18.97              | default     |
+----------------------+------------------+--------------------------------------------------------------------------------------+--------+--------------------+-------------+
mbedgt: test suite results: 83 OK
```

ccli8 commented:
### CI/ARM

```
mbedgt: test suite report:
+----------------------+------------------+-----------------------------+--------+--------------------+-------------+
| target               | platform_name    | test suite                  | result | elapsed_time (sec) | copy_method |
+----------------------+------------------+-----------------------------+--------+--------------------+-------------+
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-api-analogin          | OK     | 18.47              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-api-businout          | OK     | 34.98              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-api-digitalio         | OK     | 19.27              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-api-i2c               | OK     | 20.44              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-api-interruptin       | OK     | 19.25              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-api-pwm_fall          | OK     | 28.5               | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-api-pwm_rise          | OK     | 27.16              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-api-pwm_rise_fall     | OK     | 141.39             | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-api-spi               | OK     | 19.56              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-assumptions-analogin  | OK     | 21.47              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-assumptions-digitalio | OK     | 20.19              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-assumptions-i2c       | OK     | 17.55              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-assumptions-pwm       | OK     | 18.14              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-assumptions-pwmout    | OK     | 17.53              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-assumptions-spi       | OK     | 20.65              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-concurrent-comms      | OK     | 19.67              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-concurrent-gpio       | OK     | 21.42              | default     |
| NUMAKER_IOT_M487-ARM | NUMAKER_IOT_M487 | tests-concurrent-mixed      | OK     | 20.44              | default     |
+----------------------+------------------+-----------------------------+--------+--------------------+-------------+
mbedgt: test suite results: 18 OK
```

0xc0170 reviewed (approved):
Question about inheritance - another target that just adds all files - will be there MCU_    target for each nuvoton target and also common driver (I compared some files for 451 and 480 they look very similar)

ccli8 commented:
@0xc0170 `TARGET_NUC472`/`TARGET_M451`/`TARGET_M480`, etc. are for different MCU families. Most of their driver codes are common, but we are not yet determined to merge them into one. Its influence range would be large. But for **NUMAKER_PFM_M487**/**NUMAKER_IOT_M487** targets which belong to the same  MCU family `TARGET_M480`, their code is shared like this PR.

0xc0170 commented:
> But for NUMAKER_PFM_M487/NUMAKER_IOT_M487 targets which belong to the same MCU family TARGET_M480, their code is shared like this PR.

I can see  2 definitions in targets.json (no inheritance that 2 would benefit from) ? Would it make sense to create MCU M480 that would define what both can just inherit and add own definitions on top of it?

ccli8 committed:
[M487] Create MCU_M480 for inheritance by M487 targets

In targets.json, create MCU_M480 for inheritance by NUMAKER_PFM_M487/NUMAKER_IOT_M487.

ccli8 commented:
> I can see 2 definitions in targets.json (no inheritance that 2 would benefit from) 

@0xc0170 Yes, they are duplicate. In https://github.com/ARMmbed/mbed-os/pull/8164/commits/6ffdff48f13eab776d7d7444ad1f648b605cd744, I create an intermediate target `NU_M480` for inheritance by **NUMAKER_PFM_M487**/**NUMAKER_IOT_M487** targets to decrease duplicate defiinitions.

0xc0170 commented:
Looks good. Just one - should it have prefix `MCU_` ? ( see other similar targets there).

ccli8 commented:
> should it have prefix MCU_

@0xc0170 In https://github.com/ARMmbed/mbed-os/pull/8164/commits/9d2fcdb773e04df5605e08f66dfac1d530296ca1, `NU_M480` is renamed to `MCU_M480`.

0xc0170 commented:
/morph build
